usage.md

```diff
@@ -0,0 +1,129 @@
+# Usage
```

Comment:
This is more of an Overview than Usage, WDYT?

Reply:
I've moved this back to the resources.md I had in a previous version: I feel like usage information is important, but I realize this doc is entirely centered around resources, so I've changed it back. 

"overview" feels too much like what the README contains. This doc currently feels like mix of technical overview, resource definitions, and usage examples.

Let me know what you think.

Reply:
:+1: